src/openrct2/world/Scenery.h

```diff
@@ -28,6 +29,8 @@ struct SceneryEntryBase
 {
     StringId name;
     uint32_t image;
+    SceneryObject* obj;
```

Comment:
There is no need to add this pointer you can get the name through other means. I'm about to remove this struct as well.

Reply:
This has been reverted